wellcomeml/ml/bilstm.py

```diff
@@ -52,6 +56,17 @@ def __init__(
         self.callbacks = callbacks
         self.feature_approach = feature_approach
         self.early_stopping = early_stopping
+        self.sparse_y = sparse_y
+
+    def _yield_data(self, X, Y, batch_size, shuffle=True):
```

Comment:
Does this mean that you will only use a data generator is sparse? I thought the `if self.sparse_y` could be inside _yield_data instead, before converting to dense.

Reply:
I was thinking that i can add that in the future but why not now. Very good point.

Reply:
11b36edcae3d4ca9ac9fe54da9cba35cbcfce2fd